modules/deploy/partials/linux/install-fips.adoc

```diff
@@ -29,9 +34,32 @@ NOTE: Alternatively, you could run `sudo apt install -y redpanda-fips`, which al
 
 If you wish to only use `rpk` on a FIPS host, run:
 
+[,bash]
+----
+sudo yum install -y repdanda-fips redpanda-rpk-fips
+----
+
+--
+Debian/Ubuntu::
++
+--
+To install Redpanda for FIPS compliance, run:
+
+[,bash]
+----
+sudo apt install -y redpanda-rpk-fips redpanda-fips
```

Comment:
```suggestion
sudo apt install -y redpanda-rpk-fips redpanda-fips
```

We should remove the `-y` as it accepts all prompts. User most knowingly accept them before continuing.